### PR TITLE
Check if server is configured on web app init and forward to onboarding

### PR DESCRIPTION
**Pull request by nakhbari**
Issue: #784 

Point to the ERB onboarding for now while we're waiting on the webapp oboarding to be complete
- [x] I have run `rspec` and corrected all errors
- [x] I have run `rubocop` and corrected all errors
- [x] I have tested this change locally and tried to launch the server as well as access a project, and with that at least one build
- [x] If there is an existing issue, make sure to add `Fixes ...` as part of the PR body to reference the issue you're solving

### CODE REVIEW

File: app/features-json/setup_json_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require_relative "api_controller"
+
+module FastlaneCI
+  # Controller for providing all setup APIs
+  class SetupJSONController < APIController
+    HOME = "/data/setup"
+
+    get "#{HOME}/configured", authenticate: false do
+      return json(Services.onboarding_service.correct_setup?)
```
Comment (nakhbari) [inline app/features-json/setup_json_controller.rb]: it's just a boolean, so i'm not sure we actually need the `json()` method. Please advise.

Comment (taquitos) [reply app/features-json/setup_json_controller.rb]: Let's actually return a real object here, maybe a `ConfigurationData`. That way we can add more configuration properties as needed in the future with out having to refactor.

Comment (taquitos) [reply app/features-json/setup_json_controller.rb]: Also, let's call this endpoint `"#{HOME}/configuration"`

**Comment by KrauseFx** [comment]
@taquitos with @nakhbari on vacation, we should probably take this over. Looks like we caused some merge conflicts though 😢

File: app/features-json/setup_json_controller.rb (hunk 1 above)
Comment (KrauseFx) [reply app/features-json/setup_json_controller.rb]: Does that come with any privacy/security side effects, if a user has only e.g. very basic read access to some repos? Either way, this is not a blocker - let's merge this PR as Nima is OOO right now

**Comment by KrauseFx** [comment]
Alright, I resolved the merge conflicts 👍

**Comment by KrauseFx** [comment]
Cool, so to avoid having this PR open any longer, I'm gonna make a call here and merge this. As this is an unauthenticated request, I'm in favor of exposing as little information as we can for now. So having a boolean value is okay and does what we need it for. As we implement the onboarding, we can refine or add new API endpoints 👍

**Pull request merged**